spec/syntax.md

```diff
@@ -438,6 +438,14 @@ A _key_ can be either a _literal_ value or the "catch-all" key `*`.
 The **_<dfn>catch-all key</dfn>_** is a special key, represented by `*`,
 that matches all values for a given _selector_.
 
+The value of each _key_ MUST be treated as if it were in
+[Unicode Normalization Form C](https://unicode.org/reports/tr15/) ("NFC").
+When _keys_ are passed during _pattern selection_, the _key_ values MUST
+be normalized into NFC.
```

Comment:
This should probably be in the relevant part of the formatting section, and not here.

Reply:
I think maybe having it in both is a good idea?

Reply:
At least it needs to be in Formatting. I don't object to it also being in Syntax, but this isn't really about syntax at all.

Reply:
See the commit just below, which you'll probably have things to say about 🙈

Reply:
Drop these lines, given the Formatting additions?
```suggestion
```

Reply:
Done